app/_src/production/dp-config/dpp-on-kubernetes.md

```diff
@@ -259,7 +306,7 @@ To mitigate this, we need to either
           lifecycle:
             preStop:
               exec:
-                command: ["/bin/sleep", "30"]
+                command: ["/bin/sleep", "15"]
```

Comment:
any reason we're changing this to 15?

Reply:
Yes forgot to annotate this in the PR. The termination grace period is 30 seconds by default, so this would cause the pod to go into emergency termination, which isn't the intention.

Reply:
That is to say, it has nothing to do with this sidecar feature. Just something I noticed